### PR TITLE
refactor(virtual-background): use jss instead of sass

### PR DESCRIPTION
**Pull request by c0m1t**
Convert Sass to JSS in `virtual-background` feature.

### CODE REVIEW

File: react/features/virtual-background/components/VirtualBackgroundDialog.js [hunk 1]
```diff
@@ -106,6 +108,151 @@ function _mapStateToProps(state): Object {
...
+                display: 'block'
+            },
+            '& .thumbnail-selected': {
+                objectFit: 'cover',
+                border: '2px solid #246fe5'
+            },
+            '& .blur': {
+                boxShadow: 'inset 0 0 12px #000000',
+                background: '#7e8287',
+                padding: '0 10px'
```
Comment (robertpin) [inline react/features/virtual-background/components/VirtualBackgroundDialog.js:161]: Please reuse styles. Create a class that contains all the common styles for blur classes and use that

File: react/features/virtual-background/components/UploadImageButton.js [hunk 2]
```diff
@@ -42,6 +43,28 @@ type Props = {
     t: Function
 }
 
+const useStyles = makeStyles(theme => {
+    return {
+        addBackground: {
+            marginRight: theme.spacing(2)
```
Comment (robertpin) [inline react/features/virtual-background/components/UploadImageButton.js:49]: Please add `px` unit

**Review by robertpin — CHANGES_REQUESTED** [review]

**Comment by robertpin** [comment]
@c0m1t I don't see any changes. Forgot to push?

**Comment by c0m1t** [comment]
@robertpin Oh yes I pushed it.

**Review by robertpin — APPROVED** [review]

**Pull request merged**